common/customizable_localization/00_ruler_transition_loc.txt

```diff
@@ -367,6 +367,16 @@ DeathMessageOutro = {
 		}
```

Comment:
Mages like cats, that's the lore.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105054654-ae877900-5a8b-11eb-8e3a-8421e9bc7e12.png)

</details>